canvas_modules/harness/assets/images/blank_canvas.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 22"><defs></defs><title>blank_canvas</title><path class="cls-1" d="M12.22,3.22v-3H.22v22h30v-19Zm16,17h-26v-14h26Z" transform="translate(-0.22 -0.22)"/><path class="cls-1" d="M6.79,18.18H5V8.34H6.79Z" transform="translate(-0.22 -0.22)"/><path class="cls-1" d="M11.21,18.18a3.24,3.24,0,0,1-1.83-.49,3.38,3.38,0,0,1-1.1-1.22,4.9,4.9,0,0,1-.52-1.58,10.36,10.36,0,0,1-.14-1.63,10.36,10.36,0,0,1,.14-1.63,4.9,4.9,0,0,1,.52-1.58,3.12,3.12,0,0,1,2.93-1.71,3.12,3.12,0,0,1,2.92,1.71h0a5.17,5.17,0,0,1,.53,1.58,10.28,10.28,0,0,1,0,3.26,5.17,5.17,0,0,1-.53,1.58A3.38,3.38,0,0,1,13,17.69,3.2,3.2,0,0,1,11.21,18.18Zm0-8.12a1.06,1.06,0,0,0-.71.22,1.66,1.66,0,0,0-.46.65,4.15,4.15,0,0,0-.27,1,10.38,10.38,0,0,0-.08,1.29,10.55,10.55,0,0,0,.08,1.3,4.07,4.07,0,0,0,.27,1,1.66,1.66,0,0,0,.46.65,1.06,1.06,0,0,0,.71.22,1,1,0,0,0,.69-.22,1.59,1.59,0,0,0,.47-.65,3.69,3.69,0,0,0,.27-1h0a10.55,10.55,0,0,0,.08-1.3A10.38,10.38,0,0,0,12.64,12a3.86,3.86,0,0,0-.27-1,1.59,1.59,0,0,0-.47-.65A1,1,0,0,0,11.21,10.06Z" transform="translate(-0.22 -0.22)"/><path class="cls-1" d="M21.87,18.18a3.2,3.2,0,0,1-1.82-.49A3.38,3.38,0,0,1,19,16.47a4.91,4.91,0,0,1-.53-1.58,10.28,10.28,0,0,1,0-3.26A4.91,4.91,0,0,1,19,10.05a3.3,3.3,0,0,1,5.85,0,4.9,4.9,0,0,1,.52,1.58h0a9.56,9.56,0,0,1,0,3.26,4.9,4.9,0,0,1-.52,1.58,3.38,3.38,0,0,1-1.1,1.22A3.24,3.24,0,0,1,21.87,18.18Zm0-8.12a1,1,0,0,0-.7.22,1.66,1.66,0,0,0-.46.65,3.67,3.67,0,0,0-.27,1,10.38,10.38,0,0,0-.08,1.29,10.55,10.55,0,0,0,.08,1.3,4,4,0,0,0,.26,1,1.79,1.79,0,0,0,.47.65,1.22,1.22,0,0,0,1.4,0,1.59,1.59,0,0,0,.47-.65,4.4,4.4,0,0,0,.27-1,10.55,10.55,0,0,0,.08-1.3A10.38,10.38,0,0,0,23.31,12a4.63,4.63,0,0,0-.27-1,1.68,1.68,0,0,0-.47-.65A1,1,0,0,0,21.87,10.06Z" transform="translate(-0.22 -0.22)"/><path class="cls-1" d="M17.46,18.18H15.62V8.34h1.84Z" transform="translate(-0.22 -0.22)"/></svg>
```

Comment:
Why are the blank canvas changes part of this PR?  Is that expected?

Reply:
I bundled it all together into 1 PR so that I could test the blank canvas changes against the new dark themed background, the text and image were not properly appearing in dark theme.